app/services/imports/lettings_logs_field_import_service.rb

```diff
@@ -36,6 +38,29 @@ def update_offered(xml_doc)
       end
     end
 
+    def update_creation_method(xml_doc)
+      old_id = meta_field_value(xml_doc, "document-id")
+      log = LettingsLog.find_by(old_id:)
+
+      if log
+        upload_method = meta_field_value(xml_doc, "upload-method")
+
+        case upload_method
+        when "Manual Entry"
+          @logger.info "lettings log with old id #{old_id} entered manually, no need for update"
+        when "Bulk Upload"
+          if log.creation_method_bulk_upload?
+            @logger.info "lettings log #{log.id} creation method already set to bulk upload, no need for update"
+          else
+            log.creation_method_bulk_upload!
+            @logger.info "lettings log #{log.id} creation method set to bulk upload"
+          end
+        end
+      else
+        @logger.warn "lettings log with old id #{old_id} not found"
+      end
```

Comment:
Would a guard clause at the top instead of this else be more readable?

Reply:
I considered that and then followed the model of the other methods in these classes. I kind fo think they're equal, but it does reduce one level of nesting

Reply:
Would `return @logger.warn "lettings log with old id #{old_id} not found" unless log` not work?